cap-fs-ext/src/dir_ext.rs

```diff
@@ -157,6 +169,18 @@ pub trait DirExtUtf8 {
     fn open_dir_nofollow<P: AsRef<str>>(&self, path: P) -> io::Result<Self>
     where
         Self: Sized;
+
+    /// Removes a file or symlink from a filesystem.
+    ///
+    /// Removal of symlinks has different behavior under Windows - if a symlink points
+    /// to a directory, it cannot be removed with the remove_file operation. This
+    /// method will remove files and all sy mlinks
+    ///
+    /// This corresponds to [`std::fs::remove_file`], but only accesses paths
+    /// relative to `self`.
```

Comment:
Here and above, the "This corresponds to" sentences aren't needed here since these don't directly correspond to things in std.

Reply:
e6cb016